### ISSUE TITLE
Generate internal tags for language packs extensions

### ISSUE THREAD

**Issue by sandy081** [Microsoft/vscode-vsce#236]
With 1.20 release we support language pack extensions. If VSCE can generate internal language pack tags, it would be helpful to query them for recommendations.

Here is the contribution point schema for language packs:

``` json
languageId: string,
languageName: string,
languageNameLocalized: string,
translations: {
  'id': string, // vscode or extension id
  'path': string
}
```

Generating internal tags for languageId & extension id will be helpful.

**Comment by sandy081** [comment]
@joaomoreno I am happy to contribute here :)

**Comment by joaomoreno** [comment]
@sandy081 Sure. Do we already know which format we want?

**Comment by sandy081** [comment]
No, I can come up with some proposals.

**Comment by sandy081** [comment]
@dbaeumer Here are the tags that VSCE can generate for a lang pack extension with following contribution

```
localizations: [{
					languageId: 'de',
					translations: [{ id: 'vscode' }, { id: 'vscode.go' }]
				}]
```

- `localization-de`: Tag for languageId
- `__localization-de_vscode`, `__localization-de_vscode.go`: Internal tag for each extension translation

**Comment by sandy081** [comment]
After discussing with @dbaeumer we finalised to have languageId prepended by `lp` instead of `localization`.

**Comment by joaomoreno** [comment]
Fixed by #241, released in 1.37.0
